scripts/linters/pylint_extensions_test.py

```diff
@@ -2590,3 +2590,150 @@ def test_file_overview_at_end_of_file(self):
 
         with self.checker_test_object.assertAddsMessages(message):
             temp_file.close()
+
+
+class NewlineBelowClassDocstringTests(unittest.TestCase):
+
+    def setUp(self):
+        super(NewlineBelowClassDocstringTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.NewlineBelowClassDocstring)
+        self.checker_test_object.setup_method()
+
+    def test_no_newline_below_class_docstring(self):
+        node_no_newline_below_class_docstring = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(dummy_class):
+                        \"\"\"This is a docstring.\"\"\"
+                        a = 1 + 2
+                """)
+        node_no_newline_below_class_docstring.file = filename
+        node_no_newline_below_class_docstring.path = filename
+
+        self.checker_test_object.checker.visit_classdef(
+            node_no_newline_below_class_docstring)
```

Comment:
Why do we have to specify the `visit_classdef` in the tests? I think test should be unaware of how this is written internally, right?

Reply:
For pylint check, we need to use the method we used when writing check.

Reply:
I see, thanks for the info. I think this will help us to write a unit tests for each block. Thanks!